internal/config/load.go

```diff
@@ -81,9 +77,28 @@ func read(fs afero.Fs, path string, name string) (*viper.Viper, error) {
 	return v, nil
 }
 
-// mergeAll merges remotes and extends from .lefthook and .lefthook-local.
+func readOne(fs afero.Fs, path string, names []string) (*viper.Viper, error) {
+	for _, name := range names {
+		v, err := read(fs, path, name)
+		if err != nil {
+			var notFoundErr viper.ConfigFileNotFoundError
+			if ok := errors.As(err, &notFoundErr); ok {
+				continue
+			} else {
+				return nil, err
+			}
+		}
+
+		return v, nil
+	}
+
+	return nil, NotFoundError{fmt.Sprintf("No config files with names %q could not be found in \"%s\"", names, path)}
+}
+
+// mergeAll merges (.lefthook or lefthook) and (extended config) and (remote)
+// and (.lefthook-local or .lefthook-local) configs.
 func mergeAll(fs afero.Fs, repo *git.Repository) (*viper.Viper, error) {
-	extends, err := read(fs, repo.RootPath, "lefthook")
+	extends, err := readOne(fs, repo.RootPath, []string{".lefthook", "lefthook"})
```

Comment:
I'd like to make dot variant (`.lefthook.yml`) just an optional supported one. So, let's change this line to check `lefthook.yml` first.
```suggestion
	extends, err := readOne(fs, repo.RootPath, []string{"lefthook", ".lefthook"})
```